$MirOS: ports/print/cups/patches/patch-cups_ipp-support_c,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- cups/ipp-support.c.orig	Wed Mar  8 13:21:46 2006
+++ cups/ipp-support.c	Tue May  9 17:54:02 2006
@@ -188,7 +188,7 @@ ippErrorString(ipp_status_t error)	/* I 
   * No, build an "unknown-xxxx" error string...
   */
 
-  sprintf(cg->ipp_unknown, "unknown-%04x", error);
+  snprintf(cg->ipp_unknown, sizeof(cg->ipp_unknown), "unknown-%04x", error);
 
   return (cg->ipp_unknown);
 }
